### DISCUSSION TITLE
Sound issue when sharing screen

### DISCUSSION THREAD

**Discussion by PaulAspinall** [AntonyCorbett/OnlyM#408] [Audio Problems]
Hi, 
On The midweek meeting, everything was fine. Since then I have updated Zoom and updated OnlyM and now the sound is distorting when I play the songs on a screen share. 
coukd there be a connection with the .net download that I also had to download? 
I have uninstalled and reinstalled Zoom and the problem is still there.

**Comment by AntonyCorbett** [comment]
@PaulAspinall How are you playing the songs?

**Reply by PaulAspinall** [reply]
Through JW library. I have tested a video through windows media player and although the sound comes through my speakers fine, what other devices hear is distortion

**Reply by AntonyCorbett** [reply]
Perhaps the sound driver has been changed recently during a Windows update. I doubt this is related to OnlyM or .NET runtime. I would rollback the audio / video device driver to start with

**Reply by PaulAspinall** [reply]
Is that relatively easy to do

**Comment by Chipperchap** [comment]
Excuse me for commenting but one thing that may be of interest: We have a co-host with a similar issue. When he plays the songs, they sound like they are coming through a radio that is occasionally out of region / aerial coverage. We don't use OnlyM anymore for playing music / songs / videos but the native ZOOM video player (Share screen -> Advanced -> Video -> Share -> Locate file -> Open, when the black box appears click on the Play button, adjust the RHS sound slider to 3/4.).  (We use OnlyM for PT images though). All I am saying is that it might not be an OnlyM issue. I am going to suggest that our co-host downloads instead the 64bit version of ZOOM (everyone uses the 32bit, as do I) to see if that helps as it should be able to use better his PC resources.  If we get a good result I will share.

**Comment by timjobutler** [comment]
I too have noticed this issue at our meetings but havew not found a way to document it so that I can create a ticket with Zoom. Our AV brothers use several different programs (JW Library, OnlyM, VLC) but the issue is consistent. Like @Chipperchap said "When he plays the songs, they sound like they are coming through a radio that is occasionally out of region / aerial coverage".

I thought it was my laptop speakers but listening with a different laptop and our tablets, the same poor audio was being heard. I am running Zoom **64-bit** version 5.7.4 and the issue still persists. Personally, it sounds like Zoom is overdriving the sound levels and the music output is distorted but not sure how to go about proving that.

Happy for suggestions.

Your brother,
Tim

**Reply by stepstra** [reply]
We experienced the same problem. I found a post by another brother on reddit. His solution seems to work; tested today. Simply downgrade Zoom to Version 5.5.4 - worked for us at least.

**Reply by timjobutler** [reply]
Hi @stepstra,

Just to confirm, the only change you made was downgrading Zoom to version 5.5.4? Is your sample rate 24 bit, 48000 Hz?

**Reply by stepstra** [reply]
Actually, one of our media servants did the downgrade. Last week, he had the issue, now he didn't anymore. We never changed his sample rate.

**Comment by ColinAAvey** [comment]
This is a long shot and a complete guess, as we are not experiencing the problem here.

It may be worth trying different sample rates on the Speaker Device settings. The distortion may be related to various upsampling by Windows and down sampling by Zoom. Our songs are encoded @ 44100hz, although just to add to  the fun, other videos like the convention seem to be encoded at 48000. Screen shots below.

I would also try various settings in Zoom for "Suppress background noise" and "Music and Professional Audio".

This is based on the following fix to a similar issue in Linux 
https://superuser.com/questions/1580376/zoom-and-pulseaudio-audio-distortion-on-audio-run-through-zoom-combine

I would also try removing any virtual audio device drivers such a VB-Cables if the above doesn't help.

HTH
Colin

![image](https://user-images.githubusercontent.com/46714776/128682266-2dca5c31-7e26-40cd-acb7-afdfabf09334.png)
![image](https://user-images.githubusercontent.com/46714776/128682338-a91e2156-9e5d-4deb-bf7e-a41dcb74e63f.png)
![image](https://user-images.githubusercontent.com/46714776/128684645-0148deed-f3f5-4da1-bce0-babee71bcc1e.png)

**Reply by PaulAspinall** [reply]
That seems to work! 
I had it on 16 bit 48000 Hz and it was distorted. Now I have tried 24 bit 48000 Hz and it seems fine now. Hopefully that’s the answer then. Thank you everyone.

**Reply by timjobutler** [reply]
@PaulAspinall Just to clarify, the only adjustment made was changing from 16-bit to 24-bit?

I just want to confirm as @ColinAAvey's resolution involved changing the sample rate from 48000 to 44100 Hz.

Thanks

**Reply by ColinAAvey** [reply]
@PaulAspinall thanks for the update. One for the ever-growing checklist.

I suspect the exact configuration will vary from machine to machine, but it gives us something else to check. The worst case is that it could vary depending on the encoding of the specific media file.

**Reply by timjobutler** [reply]
@ColinAAvey I have made the changes to my Zoom settings as per your screenshot and see if that makes a different. If that doesn't work, then I will cycle through the sample rates. My **current setting is 24 bit, 48000 Hz**. My other rates are:

- 16 bit, 44100 Hz
- 16 bit, 48000 Hz
- 24 bit, 44100 Hz

As noted by @ColinAAvey, meeting songs are encoded at 44100 Hz while media videos are encoded at 48000 Hz. I only seem to notice the audio distortion during the songs which leads to believe that a switch to 44100 Hz might assist based on my current settings.

I will post more updates after my meeting this week.

![speaker-properties](https://user-images.githubusercontent.com/37101212/128711906-a9ce4c03-7ac3-43e9-89d8-b1acdd5b7f0d.png)

**Reply by PaulAspinall** [reply]
Yeah ive only changed the bit rat and that seems to have worked for the songs. I will look at changing from 48 to 44 to see if that makes any further difference/improvements.

**Comment by Chipperchap** [comment]
Hi all
So, I finally had a chance to 'quick assist' remote desktop the co-host and we found that his sound Hz settings were set and greyed out.  However, I immediately noticed that he had 'signal enhancements' -> audio enhancements enabled. So, we disabled, and tried the songs over ZOOM with and without it enabled and found that without enhancement they were playing properly without the distortion. If this is an easy fix without having to try figure out why his Hz settings were greyed out, then that is a win. Initially after changing this setting his laptop did give some warning about speakers not found or some-such but it seemed to go away and work just fine.
We have our midweek meeting Thursday so we will see what it is like in a 'real world' videoconference setting, but it looks promising. 
I will post results.
[![Audio-Enhancements.png](https://i.postimg.cc/B6gm7M1X/Audio-Enhancements.png)](https://postimg.cc/ZBWPCPHS)

**Reply by PaulAspinall** [reply]
I’ve just checked the my and the distortion is now gone. Interestingly I also changed the settings back to 16 bit and 48000 Hz and it’s fine. It looks like it may just need moving from 1 setting to another and then back again and it may solve it. 
I will try the midweek meeting on 24 bit and 48000 Hz and see if that improves the quality for those listening.

**Reply by PaulAspinall** [reply]
Although my speakers now do appear to be quieter. It may be 1 if the many the setting changes I made before changing the bit rate

**Comment by Chipperchap** [comment]
Okay so I can report that, disappointingly, nothing has changed. This is based on last Thursday and this Sunday (today). We unchecked the 'audio enhancements' button and that did not matter during a meeting - all songs were slightly distorted. We changed to 16 bit 44100 and all songs were slightly distorted. I thought that my system was okay only to be told that mine was also distorting and similar to what we hear the others are ‘broadcasting’. We are using the latest version of ZOOM 5.7.5 (939). When there is only a couple of us on it is okay, but by the time we get full attendance, around 50 participants, then the music deteriorates. Next I suppose is going back to using VLC instead of the ZOOM video player (not sure if that will make any difference) until ZOOM puts out a patch. I am a bit reluctant to go back to an earlier version of ZOOM as we can just put up with the situation for now…

**Reply by CraigBures** [reply]
@Chipperchap I can verify everything you just said. Yes, if just a few brothers are on for testing, everything sounds perfect. It's only when we go live and have many connections that the issues surfaces. This makes it difficult to test. We, too, are using the baked in Zoom video player. But if we use OnlyM to play the song video, the issue persists. So I'm not sure if using VLC will help. Please let me know if it does. This has not always been a problem. Only for about a few months I'm guessing. IMHO, this situation has been caused by a Zoom update. As @stepstra mentioned, downgrading seemed to work for him. I might give that a try just to verify. However, I agree that although downgrading may solve the sound issue, it's not the best overall solution. But it does seem to indicate that Zoom broke something. We have also tried all variations of Zoom's "Original Sound" feature, as well as the other ideas posted here. Although the issue seems to be a bit reduced, none of this has totally solved the issue. We've been hoping that Zoom would have recognized the issue by now and solved it with another update. It is interesting that the issue only seems to be present in the song videos, and none of the "human" videos.
I must say that I'm glad I found this thread. For awhile, I thought I was going crazy! LOL. I will keep an eye on this thread for any possible solutions.

**Reply by timjobutler** [reply]
Based off this input, I am going to lodge a [support ticket](https://support.zoom.us/hc/en-us/requests) with Zoom as the issue seems to be possibly related to bitrate and number of connections. The issue persists despite the media player and as @stepstra mentioned, downgrading eliminated the distortion.

Perhaps if more users lodge tickets with Zoom, it might generate some attention.

**Reply by stepstra** [reply]
Same here: testing with a few brothers does not do any good because it's always fine then. But with a few more brothers logged in the problem comes back. And from my technical understanding, setting the bit rates does only affect the output on the receiver's end but cannot change anything for the uploader.
@CraigBures Same feeling here. I tried so many things and asked so many people - I'm glad I found this thread. :-)

**Comment by ColinAAvey** [comment]
Just trying to think of the people at Zoom who may be trying to figure what the root cause is and what information could be useful. This may also limit the number of “have you tried” responses. If you can think of anything I have missed, please chip in 😊

Computer details
•	Processor
•	Graphics card and driver version
•	Sound card and drivers and which options you have tried in the speaker properties.
•	Windows version

Zoom settings
•	What settings have you tried under Audio “Suppress background noise”
•	What settings have you tried under Audio “Music and Profesional Audio”
•	What settings have you tried under the Audio.Advanced tab
•	What settings have you tried under Share Screen “Window size when screen sharing:”
•	What settings have you tried under Share Screen.Advanced tab
•	When sharing 
o	did you share  a screen, window or the media directly via advanced sharing.
o	which options did you tick (Share sound (mono), Share sound (stereo), Optimize for video clip)
o	did you share a screen of window
	which screen did you share or which screen was the window you shared on
o	if not playing the media directly, what app was used to play the media (JWLib, OnlyM, VLC etc)

Details of media which experiences the problem.
•	Video Width & Height
•	Video Data rate
•	Audio rate
•	Audio channels
•	Audio sample rate

Details of media which works.
•	As above

Details of the problem
•	Only happens over so many connections
•	Do all participants experience the problem 
•	Does the host hear the problem on their own speakers
•	Who is unmuted

**Comment by CraigBures** [comment]
So I opened a support ticket with Zoom. I will post here with whatever they have to say. I must admit that my ticket was not quite as comprehensive as @ColinAAvey suggested (lol) but I tried to give them as much as I could.

**Reply by ColinAAvey** [reply]
Sorry, too many years in IT support.

**Comment by LvDaniel** [comment]
Guys, do you how I can get an older Zoom version? On their site I can see only the latest version 5.7.5 (939).
Version 5.7.4 or older should not be affected, right?
Thank you!

**Reply by LvDaniel** [reply]
I just read all the thread an noticed that version 5.5.4 has been tested and works ok.
This is a pretty old version, released in March if I'm not wrong.
But I believe Zoom will force the upgrade to the new version.
Is there any way to prevent this?
Thank you!

**Comment by AntonyCorbett** [comment]
I'll freeze this post soon and ultimately delete it since most of the content is not associated with OnlyM and it can cause confusion when searching the discussions area. If someone knows of another repo it would be better suited to (or would like to set one up) then I can transfer the discussion there quite simply. Thanks, Antony

**Reply by PaulAspinall** [reply]
Thank you so much for allowing this chat to get to this point. It has certainly helped to raise that this is a wider issue and to seek a solution. Thank you
